src/services/eventsAPI.js

```diff
@@ -0,0 +1,94 @@
+const baseUrl = "http://localhost:3000/events";

```

Comment:
Calling to new service (locally) deployed, check https://github.com/ivan-ravecca/nodejs-postgres-api